app.js

```diff
@@ -1,16 +1,28 @@
 const express = require("express");
-const { getTopics, getEndpoints } = require("./controllers/topics.controller");
+const {
+  getTopics,
+  getEndpoints,
+  getArticlesById,
+} = require("./controllers/topics.controller");
 const app = express();
 
 app.use(express.json());
 
-app.get("/api", getEndpoints)
+app.get("/api", getEndpoints);
 
 app.get("/api/topics", getTopics);
 
+app.get("/api/articles/:article_id", getArticlesById);
+
 //err handling
 app.all("*", (req, res, next) => {
   res.status(404).send({ msg: "Route Not Found" });
+  next();
+});
+
+app.use((err, req, res, next) => {
```

Comment:
Take a look at the error handling notes as you have this here and then an overly specific catch block in your controller. You should be sending an error object to next so that your catch blocks can be generic and then this function takes that object and sends the specific promise reject that you've written.